scripts/start.sh

```diff
@@ -5,11 +5,23 @@ set -ex
 # used for loading correct manifest
 export COMMIT_HASH=`cat COMMIT_HASH.txt`
 
-if [ "$NODE_ENV" = "production" ]; then
-  node --max_old_space_size=4096 ./src
-else
+function version {
+  printf "%03d%03d%03d" $(echo "$1" | tr '.' ' ')
+}
+
+OPT=(--max_old_space_size=4096)
+
+if [ "$NODE_ENV" != "production" ]; then
   if [ ! -f "./.env" ]; then
     echo -e "\033[1;31m WARNING: Missing .env file, see CONTRIBUTING.md. \033[0m"
   fi
-  node -r dotenv/config --max_old_space_size=4096 ./src
+  OPT+=(-r dotenv/config)
+  if [ "$TERM_PROGRAM" == "vscode" ] && \
+     [ ! -z "$TERM_PROGRAM_VERSION" ] && \
+     [ $(version $TERM_PROGRAM_VERSION) -ge $(version "1.22.2") ]
```

Comment:
Check if the vscode version is >= 1.22.2 which introduced the auto-attach feature